src/platform/forms-system/src/js/web-component-patterns/relationshipToVeteranPattern.jsx

```diff
@@ -5,16 +5,21 @@ export const relationshipToVeteranUI = personTitle => {
   const person = personTitle ?? 'veteran';
 
   return {
-    relationshipToVeteran: radioUI({
-      title: `What’s your relationship to the ${person}?`,
-      labels: {
-        spouse: `I’m the ${person}’s spouse`,
-        child: `I’m the ${person}’s child`,
-        parent: `I’m the ${person}’s parent`,
-        executor: `I’m the ${person}’s executor or administrator of estate`,
-        other: 'We don’t have a relationship that’s listed here',
+    relationshipToVeteran: {
+      ...radioUI({
+        title: `What’s your relationship to the ${person}?`,
+        labels: {
+          spouse: `I’m the ${person}’s spouse`,
+          child: `I’m the ${person}’s child`,
+          parent: `I’m the ${person}’s parent`,
+          executor: `I’m the ${person}’s executor or administrator of estate`,
+          other: 'We don’t have a relationship that’s listed here',
+        },
+      }),
+      'ui:errorMessages': {
+        required: `Please enter your relationship to the ${person}`,
```

Comment:
@Thrillberg Maybe this could be "Please **select** your relationship to the person"?

Reply:
@ndsprinkle I agree that seems better. [The ticket lumps both this and the `other` relationship into one message](https://app.zenhub.com/workspaces/vagov-product-team-forms-634853151f5c6000165942bc/issues/gh/department-of-veterans-affairs/va.gov-team-forms/312) but it doesn't need to be that way.

@tbaker1026 How do you feel about Nick's suggestion for the radio selection where they select from 5 options and having your text ("Please **enter** your relationship...") for the `other` box where they are typing text in?

Reply:
Ah sorry, I didn't see you the screenshots you were going on already.

Reply:
I think we do need 2 different messages here.

The first would be "Please select your relationship to the ${person}"  And that would be the error for the first required section. And then if the user selects the last option, and doesn't fill out the text field,  the error message on the text field would be "Please enter your relationship to the ${person}"